src/test/java/lab4/game/BoardTest.java

```diff
@@ -0,0 +1,95 @@
+package lab4.game;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static lab4.game.Board.Status.*;
+
+
+
+
+public class BoardTest {
+
+    @Test
+    void testBoardIsEmpty() {
+        var board = new Board();
+        assertEquals(InProgress, board.getStatus());
+
+    }
+
+    @Test
+    void testPlaceX() {
+        var board = new Board();
+        Position pos = new Position(Row.Top, Col.Left);
+        board.placeX(pos);
+        assertTrue(board.isOccupiedAt(pos));
```

Comment:
It would be worth having a test like `assertFalse(board.isOccupiedAt(pos))` for an **unoccupied** spot as well